src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,19 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private double bladeWidth;
+    private double bladeCapacity;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(double bladeWidth, double bladeCapacity, String name, String color) {
+        super.setName(name);
+        super.setColor(color);
+        this.bladeWidth = bladeWidth;
+        this.bladeCapacity = bladeCapacity;
```

Comment:
Consider using 'this' keyword to set the fields of the current object for consistency, even though it's not strictly necessary for 'name' and 'color' here since they are accessed through superclass methods.